include/boost/geometry/core/cs.hpp

```diff
@@ -65,13 +65,13 @@ struct coordinate_system_units
 };
 
 template <>
-struct coordinate_system_units<geometry::degree>
+struct define_angular_units<geometry::degree>
 {
     typedef geometry::degree units;
 };
 
 template <>
-struct coordinate_system_units<geometry::radian>
+struct define_angular_units<geometry::radian>
```

Comment:
Right, I see. We don't get backward compatibility issues with this?

OK, I see that it is part of core_detail - then it should be no problem